LICENSE

```diff
@@ -199,4 +199,64 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+   
+   
+=======================================================================
+Apache Pirk (incubating) Subcomponents:
+
+The Apache Pirk project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
+
+========================================================================
+BSD-style licenses
+========================================================================
+
+The following components are provided under a BSD-style license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
```

Comment:
I don't see the ```licenses/``` directory being included in this PR?

We really do need to include the actual text of the 3rd-party dependency licenses in our source distribution.
See  http://www.apache.org/dev/licensing-howto.html#permissive-deps.

It's not quite one for _each_ of the Maven dependencies, things like Scala {compiler, library, parser, xml} are covered by a single license.

For an example, see Spark's [LICENSE](https://github.com/apache/spark/blob/master/LICENSE) and [licenses/ directory](https://github.com/apache/spark/tree/master/licenses).